acceptance/launcher_test.go

```diff
@@ -1,3 +1,5 @@
+// +build acceptance
```

Comment:
This seems to be required in order to share functions such as `buildTestImage` that are defined in acceptance_test.go. I am not sure why it was missing before.